frontend/src/modules/Servicemap/utils.ts

```diff
@@ -59,6 +59,7 @@ export const getGraphData = (serviceMap, isDarkMode): graphDataType => {
 				width: MIN_WIDTH,
 				color,
 				nodeVal: MIN_WIDTH,
+				name: node,
```

Comment:
We do not need this `name` property, as property `id` is pointing to the same.

Reply:
Without the `name` property, the [on hover tooltip](https://private-user-images.githubusercontent.com/8893755/295887988-7cb7f061-a961-416a-b8cb-8772333cb88b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5OTUyOTgsIm5iZiI6MTcwNTk5NDk5OCwicGF0aCI6Ii84ODkzNzU1LzI5NTg4Nzk4OC03Y2I3ZjA2MS1hOTYxLTQxNmEtYjhjYi04NzcyMzMzY2I4OGIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMjNUMDcyOTU4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGZiNzRkOTBkYjI5NzE0MWNiNGM4YTU4MmEwNmNjOTI2ZWYwNTdkMmQ1MDlkNDUxMDc3YWI3NjUzYjRkMzI5YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.175rQDtrvZwAAE6ME1IN-UNDKgaLqmge5ccz2knrOYw) doesn't work.